pkg/hubble/observer/local_observer.go

```diff
@@ -283,6 +283,15 @@ nextEvent:
 		switch ev := e.Event.(type) {
 		case *flowpb.Flow:
 			eventsReader.eventCount++
```

Comment:
@tklauser do you know why `eventCount` is only incremented when the event is a flow (i.e. it's not incremented for a `LostEvent` for example)?

Reply:
`LostEvent` is a special case and if you need to ask then it probably deserves a comment there in the switch case. A `LostEvent` is an event type that is never explicitly requested by the user (say, a query asks for 20 events, then lost events should not be accounted for as they are not events per se but an indication that some event was lost). The Hubble CLI actually prints lost events to stderr iirc.
You can also have a look at the comments and LostEvent being treated as a special case in `newRingReader()` and `eventsReader.Next()` further below.

Reply:
Thanks for clarifying @rolinh!

Reply:
Thanks for the explanation @rolinh, I've added a comment in the LostEvent case.